docs/schema.md

```diff
@@ -844,6 +847,55 @@ for sufficient available resources, and new clusters will be created with
 the required amount of resources (higher than the stated requirement, as
 not all cluster resources are available for workload use).
 
+#### GPUs
+
+An app may request at least one node with one or more GPUs with a supported,
+compatible platform. For instance, the following specifies that at least one
+Nvidia GPU must exist in the cluster:
+
+```yaml
+x-google-marketplace:
+  clusterConstraints:
+    resources:
+    - requests:
+        gpu:
+          nvidia.com/gpu: {}
+```
+
+The following specifies that at least one node in the cluster should have at
+least two Nvidia GPUs (with platform `nvidia-tesla-k80` or `nvidia-tesla-k100`):
+
+```yaml
+x-google-marketplace:
+  clusterConstraints:
+    resources:
+    - requests:
+        gpu:
+          nvidia.com/gpu:
+            limits: 2
+            platforms:
+            - nvidia-tesla-k80
+            - nvidia-tesla-k100
+```
+
+Currently only Nvidia platforms are supported (requests entry key
+`nvidia.com/gpus`). See the
+[Compute Engine GPU guide](https://cloud.google.com/compute/docs/gpus)
+for a full list of available GPU platforms.
+
+Note that for `resource` entries with GPU requests:
+
+*   `resources` may contain at most one entry with GPU requests
+*   `affinity` and `replicas` are ignored
+    * `cpu` and `memory` may be defined in the same entry, but must be
```

Comment:
Rm. `cpu` and `memory` actually need replicas, otherwise we don't know how to fit.

Reply:
Done.